core/services/keeper/registry_synchronizer_core.go

```diff
@@ -62,13 +65,15 @@ type RegistrySynchronizer struct {
 // NewRegistrySynchronizer is the constructor of RegistrySynchronizer
 func NewRegistrySynchronizer(opts RegistrySynchronizerOptions) *RegistrySynchronizer {
 	mailRoom := MailRoom{
-		mbUpkeepCanceled:    utils.NewMailbox[log.Broadcast](500),
-		mbSyncRegistry:      utils.NewMailbox[log.Broadcast](1),
-		mbUpkeepPerformed:   utils.NewMailbox[log.Broadcast](3000),
-		mbUpkeepRegistered:  utils.NewMailbox[log.Broadcast](500),
-		mbUpkeepReceived:    utils.NewMailbox[log.Broadcast](500),
-		mbUpkeepMigrated:    utils.NewMailbox[log.Broadcast](500),
-		mbUpkeepGasLimitSet: utils.NewMailbox[log.Broadcast](500),
+		mbUpkeepCanceled:         utils.NewMailbox[log.Broadcast](500),
+		mbSyncRegistry:           utils.NewMailbox[log.Broadcast](1),
+		mbUpkeepPerformed:        utils.NewMailbox[log.Broadcast](3000),
+		mbUpkeepRegistered:       utils.NewMailbox[log.Broadcast](500),
+		mbUpkeepReceived:         utils.NewMailbox[log.Broadcast](500),
+		mbUpkeepMigrated:         utils.NewMailbox[log.Broadcast](500),
+		mbUpkeepGasLimitSet:      utils.NewMailbox[log.Broadcast](500),
+		mbUpkeepPaused:           utils.NewMailbox[log.Broadcast](500),
```

Comment:
we'll also need to listen to Unpause event

Reply:
Done